pkg/webhook/webhook.go

```diff
@@ -117,7 +108,7 @@ func (pwh *pravegaWebhookHandler) mutatePravegaVersion(ctx context.Context, p *p
 	if err != nil {
 		return fmt.Errorf("request version is not in valid format: %v", err)
 	}
-	if _, ok := supportedVersions[normRequestVersion]; !ok {
+	if _, ok := versionMap[normRequestVersion]; !ok {
```

Comment:
It would be good to maintain the same reference name "supportedVersions" for the Map, so do not need to change the references everywhere.